plugins/utils/nix-develop.nix

```diff
@@ -0,0 +1,44 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+}:
+with lib; let
+  cfg = config.plugins.nix-develop;
+  helpers = import ../helpers.nix {inherit lib;};
+in {
+  options.plugins.nix-develop =
+    helpers.extraOptionsOptions
+    // {
+      enable = mkEnableOption "nix-develop.nvim";
+
+      package = helpers.mkPackageOption "nix-develop.nvim" pkgs.vimPlugins.nix-develop-nvim;
+
+      ignoredVariables = mkOption {
+        type = types.attrsOf types.bool;
+      };
+
+      separatedVariables = mkOption {
+        type = types.attrsOf types.str;
+      };
+    };
+
+  config = let
+    inherit (cfg) ignoredVariables separatedVariables;
```

Comment:
im struggling a bit with the correct syntax for the inherit ?

Reply:
I think it would be clearer if you skip the let (cfg) inherit .... in, and directly to cfg.ignoredVariables & cfg.separatedVariables in extraConfigLua